src/test/java/seedu/address/testutil/ClientBuilder.java

```diff
@@ -40,8 +42,8 @@ public class ClientBuilder {
     public ClientBuilder() {
         name = new Name(DEFAULT_NAME);
         phone = new Phone(DEFAULT_PHONE);
-        email = new Email(DEFAULT_EMAIL);
-        address = new Address(DEFAULT_ADDRESS);
+        email = Optional.of(new Email(DEFAULT_EMAIL));
+        address = Optional.of(new Address(DEFAULT_ADDRESS));
         birthday = Optional.empty();
```

Comment:
hmm, is this meant to stay an empty optional or is it meant to use default birthday?

Reply:
This issue is mainly about test cases, it should be using default birthday but I haven't added the birthdays for all the default clients yet which cause the test cases to fail, will add that in during v1.4